### ISSUE TITLE
Strong growth of the number of dependencies

### ISSUE THREAD

**Issue by RonaldAJ** [iluvcapra/wavinfo#13]
The new release shows a strong growth of the number of dependencies in requirements.txt. For our use case this is a potential problem, as we prefer to keep the number of packages installed on our microphone processing software limited. We like to install with the minimal possible set needed for running and testing. 

Also the requirements pins all the version to a specific version, rather than specifying a minimal version. 

Is it possible to limit the number of dependencies and make them more flexible?

**Comment by RonaldAJ** [comment]
As I appreciate the work done I like to keep my fork synced!

**Comment by RonaldAJ** [comment]
Most of the packages mentioned in the requirements.txt are related to ear which is only used in wave_axml_reader.py. As far as I can see  wave_axml_reader.py is not used. May be it is possible to drop both the dependency and the wave_axml_reader.py?

**Comment by iluvcapra** [comment]
I think we could afford to drop ear, I don't know how many people are using it and I find that whenever I need the ear dataclasses I just reimport it for whatever project I'm using

**Comment by iluvcapra** [comment]
So what happened was, I got a lot more fancy with my documentation in the last version and sphinx pulled a ton of packages into the venv.

In addition to removing ear, I've also created separate venvs for the project and for the documentation build and this has straightened everything out, now the only dependency in the project proper is lxml.

These changes are in commit bba4d67641a23b354b1a78a474020e5993e8cdb2 and I will draft a release for this now.
